index.html

```diff
@@ -0,0 +1,460 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <script src="https://kit.fontawesome.com/ea6955352c.js" crossorigin="anonymous"></script>
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars"></i>
+      </button>
+      <nav id="mobile-menu" class="not-active">
+        <div>
+          <i id="close" class="fas fa-xmark"></i>
+          <ul>
+            <li><a class="mobile-link" href="#projects">Protfoliu</a></li>
+            <li><a class="mobile-link" href="#aAbout-me">About</a></li>
+            <li>
+              <a class="mobile-link" href="#c#ontact">Contact</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
```

Comment:
- [ ] For some reason your mobile menu shows by default when the orientation of the page is switched to mobile view. This is not the desired effect as the user is the one who is supposed to click to open the meu and not have it open by default. Below is a screencast to further explain this situation:

[Screencast from 24-04-2023 07:03:05 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/234053359-03a3ef8e-8e03-4912-82bf-a5d62470ee18.webm)